### PR TITLE
[WIP ]New roound of animation updates

### PR DESCRIPTION
**Pull request by bhamlefty**
Updating the animation library to be able to receive keyframeParams for scales, fades, and slides.

### CODE REVIEW

File: packages/react/src/themes/teams/animations/scale.ts [hunk 1]
```diff
@@ -3,47 +3,50 @@ import { easeOut, easeIn, easeEasy } from './timingFunctions'
 const scaleAnimations = {
   // Scale in w/ Fade -- Useful for opening modal dialogs
   scaleEnterFast: {
-    keyframe: {
+    keyframe: ({ delta }) => ({
```
Comment (mnajdova) [inline packages/react/src/themes/teams/animations/scale.ts:6]: The changes looks good! Let's just extract the repeting functions in a reusable const, something like: 
```
const scaleKeyframe =  ({ delta }) => ({
  '0%': {	     
    transform: `scale(${delta})`,
    opacity: 0,
  },	     
  '100%': {	 
    transform: 'scale(1)',	   
    opacity: 1,	   
  },	     
}),
```

and then define the animations as:
```
scaleEnterFast: {
  keyframe: scaleKeyframe,
  duration: '500ms',
  timingFunction: easeOut,
  keyframeParams: { delta: '.9' },
  fillMode: 'forwards'
}
```

File: packages/react/src/themes/teams/animations/scale.ts [hunk 2]
```diff
@@ -3,47 +3,50 @@ import { easeOut, easeIn, easeEasy } from './timingFunctions'
...
       },
       '100%': {
         transform: 'scale(1)',
         opacity: 1,
       },
-    },
+    }),
     duration: '500ms',
     timingFunction: easeOut,
+    keyframeParams: { distance: '.9' },
```
Comment (mnajdova) [inline packages/react/src/themes/teams/animations/scale.ts:18]: let's decide on `distance` or `delta`, it should be the same name.

File: packages/react/src/themes/teams/animations/slide.ts [hunk 3]
```diff
@@ -188,7 +188,7 @@ const slideAnimations = {
     fillMode: 'forwards',
   },
   // Slides object in from left to right
-  SlideRightEnterMedium: {
+  slideRightEnterMedium: {
```
Comment (mnajdova) [inline packages/react/src/themes/teams/animations/slide.ts:191]: 👍

File: packages/react/src/themes/teams/animations/scale.ts (hunk 2 above)
Comment (notandrew) [reply packages/react/src/themes/teams/animations/scale.ts:18]: I vote for distance because it's more clear, in my mind.

**Comment by jdhuntington** [comment]
Closing as this has seen no activity for awhile. Please reopen if still relevant. Thanks!

**Pull request closed without merging**